exec/src/klio_exec/commands/run.py

```diff
@@ -260,11 +270,25 @@ def _get_run_callable(self):
             )
             raise SystemExit(1)
 
+    def _write_run_effective_config(self):
+        # this method assumes setup.py is being used!
+        path = config_core.RUN_EFFECTIVE_CONFIG_PATH
+        logging.debug(
+            "dumping runtime configuration to {}"
```

Comment:
Small nitpick - could it be "Writing" instead of "dumping"? Either way, I suggest titlecasing the first word as well as `Docker`. 

Also, maybe this is just me, but is a little misleading to say "in job's Docker image", since it's not in _the_ image? just the container? Maybe, `"Writing runtime configuration to {} in the job's running Docker container."`?